### DISCUSSION TITLE
Is there a pure JavaScript online version of Monaco Editor?

### DISCUSSION THREAD

**Discussion by erosman** [microsoft/monaco-editor#4772] [Q&A]
Subsequent to [How to use Monaco Editor in a browser extension?](https://github.com/microsoft/monaco-editor/discussions/3908), is there a playground type of Monaco Editor that is meant for online use?

I have seen [vscode.dev](https://vscode.dev/) but I am looking for a minimal editor with only  JavaScript & CSS support (syntax highlighting and linting).

**Comment by pradipNP** [comment]
If you're looking for a ready-made, minimal online playground rather than the full VS Code experience, I don't know of an official Monaco playground that provides exactly that JavaScript + CSS-only setup.

Monaco Editor itself can be embedded in a web page and configured to load only the languages/features you need, so a small custom playground using Monaco could provide the minimal experience you're looking for.

If you're specifically looking for an existing hosted playground, though, vscode.dev is probably closer to a full editor than what you're after.
